tests/Http/Role/GetUserPermissionsRequestHandlerTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Role;
+
+use CultuurNet\UDB3\Http\Request\Psr7RequestBuilder;
+use CultuurNet\UDB3\Http\Response\AssertJsonResponseTrait;
+use CultuurNet\UDB3\Http\Response\JsonResponse;
+use CultuurNet\UDB3\Role\ReadModel\Permissions\UserPermissionsReadRepositoryInterface;
+use CultuurNet\UDB3\Role\ValueObjects\Permission;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Slim\Psr7\Headers;
+
+class GetUserPermissionsRequestHandlerTest extends TestCase
+{
+    use AssertJsonResponseTrait;
+
+    /**
+     * @var UserPermissionsReadRepositoryInterface|MockObject
+     */
+    private $permissionsRepository;
+
+    private string $currentUserId;
+
+    protected function setUp(): void
+    {
+        $this->permissionsRepository = $this->createMock(UserPermissionsReadRepositoryInterface::class);
+    }
+
+    /**
+     * @test
+     */
+    public function it_returns_an_array_of_permissions_for_the_current_god_user(): void
```

Comment:
I think this test is for a given user instead of the current user. Same remark for `it_returns_an_array_of_permissions_for_the_current_user`

Reply:
The route is `user/permissions` so I think it does reflect the current user. Also, this case name was copied from the `ReadRoleRestControllerTest`

Reply:
You are right! I got confused by the variable `$userId`, but that is only used for the `permissionsRepository`.